Sources/XCHammer/Generator.swift

```diff
@@ -874,10 +875,15 @@ enum Generator {
         }
 
         let ruleEntryMap = workspaceInfo.ruleEntryMap
-        guard let genfileLabels = try? BazelQueryer.genFileQuery(targets:
-            config.buildTargetLabels, bazelPath: bazelPath,
-            workspaceRoot: workspaceRootPath) else {
-            fatalError("Can't get genfiles")
+        let genfileLabels: [BuildLabel]
+        do {
+            genfileLabels = try BazelQueryer.genFileQuery(targets:
+                config.buildTargetLabels, bazelPath: bazelPath,
+                workspaceRoot: workspaceRootPath)
+        } catch let e as ShellOutError {
+            fatalError("Bazel failure:\n\(e.message)")
```

Comment:
@kastiglione would you update this to return `.failure` un unsuccessful results? Here, it generally returns early `Result`s.